src/components/AssistanceForm/AssistanceForm.tsx

```diff
@@ -31,18 +29,25 @@ export const AssistanceForm = () => {
   async function getAssistanceJWT(e: React.MouseEvent<HTMLButtonElement, MouseEvent>) {
     e.preventDefault();
     try {
-      const { data: zendeskAssistance } = await utils.apiClient.token.getZendeskAssistanceToken({
-        userEmail: email
-      });
+      await utils.apiClient.token
+        .getZendeskAssistanceToken({
+          userEmail: email
+        })
+        .then((data) => {
```

Comment:
Can we remove the await and move the catch as `then(data => {}).catch(e => {})` or do the following instead?
```javascript
const data = await utils.apiClient.token.getZendeskAssistanceToken()
```